src/IECoreDelight/Renderer.cpp

```diff
@@ -1106,6 +1140,8 @@ class DelightRenderer final : public IECoreScenePreview::Renderer
 			}
 			else if( boost::starts_with( name.string(), "dl:" ) )
 			{
+				ConstDataPtr conformedValue = conformOptionValue( name, value );
```

Comment:
We're currently calling `conformOptionValue()` with a nullptr when `value` is null, which will also trigger a crash in `conformOptionValue()`. That might be fixed by my suggestion above, but I think it probably makes more sense to move this inside the `if( value )` block - it seems nonsensical to conform a null value.

Reply:
Definitely does not make sense to try and convert a null value, fixed in https://github.com/GafferHQ/gaffer/pull/5333/commits/166474356a9cc4f9db52b528a734678df939b569